platform/lib/routers/example/experiments.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+
+const {Templates, createRequestContext} = require('@lib/templates/index.js');
+const express = require('express');
+
+// eslint-disable-next-line new-cap
+const sharedPages = express.Router();
+
+sharedPages.get('/shared/experiments/', async (req, res, next) => {
+  try {
+    const template = await Templates.get('/shared/experiments.html');
+    const rendered = template.render({'experiments': req.query.experiments.split(',')});
+    res.send(rendered);
+  } catch (e) {
+    // page not found
```

Comment:
The comment seems off 😉Also pass `e` to `next` as if something goes wrong that is most probably a nunjucks/rendering error.